src/main/java/org/nrnb/gsoc/enrichment/utils/ModelUtils.java

```diff
@@ -499,6 +500,7 @@ public static List<EnrichmentTerm> getEnrichmentfromJSON(JSONObject response, Cy
         if (enrichmentArray == null) {
             return null;
         }
+        System.out.println("Node Name List: " + nodeNameList.toString());
```

Comment:
Is this for logging purposes? If not remove the statement. Same comment for all print statements

Reply:
Sorry, my bad. I forgot to remove them before making the PR. Thanks for pointing it out.